packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -54,7 +61,10 @@ export async function createNewFluidFile(
     }
 
     const filePath = newFileInfo.filePath ? encodeURIComponent(`/${newFileInfo.filePath}`) : "";
-    const encodedFilename = encodeURIComponent(`${newFileInfo.filename}.fluid`);
```

Comment:
i don't think we should do anything with file extension. it should just be passed in as part of the file name. for backwards compat we can allo appeding .fluid if none is provided

Reply:
Expecting extension in filename itself and use .fluid if not supplied.